vnext/Microsoft.ReactNative/Utils/ManagedResource.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+#include <utility>
+
+template <typename T, typename closeFn_t>
+class managed_resource {
```

Comment:
>managed_resource [](start = 6, length = 16)

We typically use Pascal casing for the class and function names. Please follow the same style for new code.

[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions)

Reply:
@vmoroz that's a dotnet framework guideline, I don't think we've been following that super closely. I think it makes sense for RNW-specific functionality, whereas for utilitarian things it might make more sense to align with library casing. See other existing examples of this in the codebase in the same directory as this file, e.g. Utils/PropertyHandlerUtils.h (`get_argument_type`). I don't feel super strongly either way, lmk if you do.